build.sbt

```diff
@@ -0,0 +1,147 @@
+import PlayArtifact._
+import sbt._
+import sbt.Keys._
+import sbtassembly.Plugin.{AssemblyKeys, MergeStrategy}
+import AssemblyKeys._
+import Dependencies._
+import sbtrelease._
+import sbtrelease.ReleasePlugin._
+import ReleaseStateTransformations._
+import xerial.sbt.Sonatype.SonatypeKeys
+import xerial.sbt.Sonatype._
+import com.typesafe.sbt.pgp.PgpKeys
+import play.sbt.routes.RoutesKeys._
+
+val scala211 = "2.11.12"
+val scala212 = "2.12.4"
+
+val commonSettings =
+  Seq(
+    scalaVersion := scala212,
+    scalaVersion in ThisBuild := scala212,
+    crossScalaVersions := Seq(scala211, scala212),
```

Comment:
I suspect we can remove 2.11 now, if not very soon.

Reply:
I have a spreadsheet (internal to the Guardian) that lists all the versions of Panda that are still out there in the wild. I'll share it with you now